tests/test_ag3.py

```diff
@@ -1829,6 +1829,102 @@ def test_h1x_gwss():
     assert np.all(h1x >= 0)
 
 
+def test_average_fst():
+    ag3 = setup_ag3()
+    region = "3L"
```

Comment:
```suggestion
    region = "3L:15,000,000-16,000,000"
```

For testing purposes, should be fine to run on a smaller genome region, which will mean tests run faster.